include/regs.h

```diff
@@ -157,9 +157,9 @@
 #define R_TEXTBOX_BG_YPOS           XREG(61)
 #define R_TEXTBOX_CLEF_XPOS         VREG(7)
 #define R_TEXTBOX_CLEF_YPOS         VREG(8)
-#define R_OCARINA_NOTES_XPOS        VREG(28)
-#define R_OCARINA_NOTES_YPOS(note)  VREG(45 + (note))
-#define R_OCARINA_NOTES_XPOS_OFFSET VREG(29)
-#define R_OCARINA_NOTES_YPOS_OFFSET VREG(51)
+#define R_OCARINA_BUTTONS_XPOS        VREG(28)
+#define R_OCARINA_BUTTONS_YPOS(note)  VREG(45 + (note))
+#define R_OCARINA_BUTTONS_XPOS_OFFSET VREG(29)
+#define R_OCARINA_BUTTONS_YPOS_OFFSET VREG(51)
```

Comment:
it's weird calling the drawn notes "buttons", though that's what they show up as, because it's inconsistent with some of the names like `sOcarinaNoteTextures` among other stuff

I don't really like "note" or "button" better, both aren't quite right to me (note is supposed to be sound and button is supposed to be input, neither mean something to be drawn)

there is probably something I'm missing about the difference between `OcarinaButtonIdx` and `OcarinaNoteIdx`

Reply:
The `noteIdx` has been renamed to `pitch` throughout, although `buttonIdx` is still open for possible renaming

Reply:
so I understand OcarinaButton/OcarinaBtn stuff can't be "note" because that's an overused term, but I don't like "button" much because then it can be confused with the controller buttons

The more I think about it, the more introducing the term `btnNote` sounds like a good idea but I'm not sure

Reply:
imo context makes it pretty clear what it is. buttons are closely tied to the ocarina system so its gonna be hard to escape that, and the second i read the context here I know what its referring to personally

Reply:
At the end of the day, what's drawn on the staff are just buttons on the controller:
![image](https://user-images.githubusercontent.com/47598039/153156807-6415e8cb-c3e6-4cab-866e-30a00db2d6f7.png)

Just like how the equips draw buttons on the top right of the screen, those are named `buttonItems`:
![image](https://user-images.githubusercontent.com/47598039/153156474-5bc0bc9f-ab37-4c02-be6e-04a3ce3333a9.png)

Which are all drawing the physical buttons on the controller:
![image](https://user-images.githubusercontent.com/47598039/153156735-5926b83e-94f9-4b7b-85d1-1be97f41ee4c.png)

They're all buttons in a way, it's just making sure the right word accompanies the name, like `buttonItem` or `ocarinaButton` (or `buttonOcarina`). Maybe `buttonNote` would be fine, although like you mentioned Note is oversaturated so I'd like to avoid it unless necessary

Reply:
I'm just worried that "button" means both `OCARINA_BUTTON_` and `BTN_`

Maybe another way to address this concern could be to, where "button" means `BTN_`, change it to `ctrlrButton`
For example "button" in `sOcarinaAllowedButtonMask`, `sOcarinaAButtonMap`, ... refers to controller button?

Reply:
Have you had a chance to think about "ocarina button" more? I'm still fine with the current name I have, but I'm not sure if you've come up with any other ideas since

Reply:
Im still of the opinion that context will help you realize what "button" is referring to, fwiw

Reply:
It's been a long time unfortunately so I forgot some of the points that were pressed,

Would "ocarina input" work in place of "ocarina button"?

Reply:
k let's just go with button 🤷